one/src/http_metrics.rs

```diff
@@ -15,34 +11,30 @@ use prometheus_client::{
     registry::Registry,
 };
 
-#[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
-struct RequestLabels {
-    path: &'static str,
-}
-
-impl From<&Event> for RequestLabels {
-    fn from(value: &Event) -> Self {
-        Self { path: value.path }
-    }
-}
-
 /// Metrics for Kubo RPC API
 #[derive(Clone)]
-pub struct Metrics {
+pub struct HttpMetrics {
     requests: Family<RequestLabels, Counter>,
     request_durations: Family<RequestLabels, Histogram>,
 }
 
 #[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
 struct InfoLabels {
-    base_path: &'static str,
-    version: &'static str,
+    api_version: &'static str,
+    kubo_api_version: &'static str,
 }
 
-impl Metrics {
+#[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
+struct RequestLabels {
+    method: String,
+    path: String,
+    status: u16,
+}
+
+impl HttpMetrics {
     /// Register and construct Metrics
     pub fn register(registry: &mut Registry) -> Self {
-        let sub_registry = registry.sub_registry_with_prefix("ceramic_api");
+        let sub_registry = registry.sub_registry_with_prefix("http_api");
```

Comment:
open to a better name (e.g. ceramic_api) but I tried to avoid the existing sub registries I was going to delete